### PR TITLE
Basic support for replacing events by id

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
Why hardcode "id"? Why not just "replace by key"?

File: aw_datastore/storages/peewee.py [hunk 1]
```diff
@@ -156,19 +160,35 @@ def insert_many(self, bucket_id, events: List[Event]):
...
+        return EventModel.select() \
+                         .where(EventModel.id == event_id) \
+                         .where(EventModel.bucket == self.bucket_keys[bucket_id]) \
+                         .get()
+
+    def _get_last(self, bucket_id) -> EventModel:
         return EventModel.select() \
                          .where(EventModel.bucket == self.bucket_keys[bucket_id]) \
                          .order_by(EventModel.timestamp.desc()) \
-                         .limit(1) \
```
Comment (johan-bjareholt) [inline aw_datastore/storages/peewee.py]: Why is this removed? Does ".get" only return one element?

Comment (ErikBjare) [inline aw_datastore/storages/peewee.py]: Yes.

**Comment by ErikBjare** [comment]
@johan-bjareholt Events in datastores already had id's. For example: an ID field is created by default by peewee, it's just explicit now (it won't break the database schema). 

The id's may vary in type depending on datastore method (integers for memory and peewee, str for MongoDB). They don't serve any purpose except as a way to specify events to edit/delete.

An optional ID field has been added to the Event class and events with id's will be returned by the server when queried.
